packages/model/db.go

```diff
@@ -400,7 +398,9 @@ func GetRollbackID(transaction *DbTransaction, tblname, where, ordering string)
 	var result int64
 	err := GetDB(transaction).Raw(`SELECT rb_id FROM "` + tblname + `" ` + where + " order by rb_id " + ordering).Row().Scan(&result)
 	if err != nil {
-		log.WithFields(log.Fields{"type": consts.DBError, "error": err}).Error(fmt.Errorf("GetRollbackID from table %s where %s order by rb_id %s", tblname, where, ordering))
+		log.WithFields(log.Fields{
+			"type": consts.DBError, "error": err,
+		}).Error(fmt.Errorf("GetRollbackID from table %s where %s order by rb_id %s", tblname, where, ordering))
```

Comment:
it is better to move query to separate variable, then log it like DBError, "query": q

Reply:
done,